### PR TITLE
ENH: Paranormal distribution

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to add paranormal distribution.

Screenshot 2021-03-31 160903

Screenshot 2021-03-31 160942

p.s. Ancient aliens are out of scope.

### CODE REVIEW

**Comment by hamogu** [comment]
I appreciate that the bot gives advice to @pllim on how to skip the CI!

**Review by eteq — CHANGES_REQUESTED** [review]
One important suggestion inline. 

 As a broader comment, maybe we should adopt this as the default for uncertainties throughout astropy.  After all, super-natural is clearly better than a natural distribution for a given problem...

File: astropy/uncertainty/distributions.py [hunk 1]
```diff
@@ -69,7 +71,120 @@ def normal(center, *, std=None, var=None, ivar=None, n_samples,
...
+        a.plot()
+
+    """
+    center = np.asanyarray(center)
+
+    # Only one allowed, as per original figure.
+    # Somehow Quantity creates a mysterious QuantityParanormalDistribution
+    # that I cannot control.
+    if center.size != 1:
+        raise ValueError('oooOOOooOOoooo')
```
[the remark this answers is not in the document]
Comment (pllim) [reply astropy/uncertainty/distributions.py]: Great idea!

**Comment by pllim** [comment]
It is unfortunate RTD is currently broken due to #11458 . So you will just have to trust the screenshots.

**Comment by dhomeier** [comment]
I just realise we really should have put _Improving Support for Spectre_ as top-level functionality bullet point in the roadmap!

**Comment by mwcraig** [comment]
> I just realise we really should have put Improving Support for Spectres 

I suppose spectres might be easier than spectra 🤔

**Comment by pllim** [comment]
Alright, we got our fun. Until next year! 😉

**Pull request closed without merging**